ui-modules/app-inspector/app/components/entity-tree/entity-tree.directive.js

```diff
@@ -101,7 +97,7 @@ export function entityTreeDirective() {
     }
 }
 
-export function entityNodeDirective() {
+export function entityNodeDirective($log, iconService) {
```

Comment:
Seems to not be necessary. The `$log` service is not used and the `iconService` is injected on the controller level

Reply:
correct.  also removed the `inject` line